test/inductor/test_max_autotune.py

```diff
@@ -52,7 +52,10 @@ def _get_path_without_sccache() -> str:
 
 
 def benchmark_choice(choice, args, out, expected_out, timings):
-    result = choice.benchmark(*args, out=out)
+    # choice.benchmark outputs a LazyBenchmark is some cases,
+    # we need to initialize the lazy value by converting it
+    # to a float
+    result = float(choice.benchmark(*args, out=out))
```

Comment:
similarly, i dont think we should change the output type by default.

Reply:
Fixed